webapi/src/test/scala/org/knora/webapi/messages/v2/responder/ontologymessages/KnoraCardinalityInfoIsStricterThanZSpec.scala

```diff
@@ -0,0 +1,82 @@
+/*
+ * Copyright © 2021 - 2022 Swiss National Data and Service Center for the Humanities and/or DaSCH Service Platform contributors.
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+package org.knora.webapi.messages.v2.responder.ontologymessages
+
+import dsp.schema.domain.Cardinality._
+import zio.test._
+
+/**
+ * Tests the comparison forstrictness of cardinalities with [[isStricterThan()]].
+ */
+object KnoraCardinalityInfoIsStricterThanZSpec extends ZIOSpecDefault {
+
+  def spec = (mustHaveOneTest + mustHaveSomeTest + mayHaveOneTest + mayHaveManyTest)
+
+  val mustHaveOne  = MustHaveOne  // "1"
+  val mustHaveSome = MustHaveSome // "1-n"
+  val mayHaveOne   = MayHaveOne   // "0-1"
+  val mayHaveMany  = MayHaveMany  // "0-n"
+
+  private val mustHaveOneTest = suite("MustHaveOneTest")(
```

Comment:
does this have to be private? I suppose so, right?

Reply:
It doesn't have to be. As private values caused us trouble with fmt, I decided to delete it here and in other ZIO specs.